public_html/wp-content/plugins/wcpt/wcpt-event/class-event-admin.php

```diff
@@ -330,16 +330,29 @@ public static function schedule_decline_notification( $event, $label, $location
 	 * @param int    $event_id
 	 * @param string $label
 	 * @param string $location
+	 *
+	 * @return null|bool|string
 	 */
 	public static function send_decline_notification( $event_id, $label, $location ) {
+		$declined_notification_key = 'Sent declined notification';
```

Comment:
It's possible to have spaces in keys, but conventionally they're `underscore_case`.